### PR TITLE
testing: RpcBehaviorLoadBalancingProvider to use acceptResolvedAddresses()

### PR DESCRIPTION
**Pull request by temawi**
Move this test LB over to using `acceptResolvedAddresses()` as `handleResolvedAddresses()` is going to be deprecated.

### CODE REVIEW

File: interop-testing/src/test/java/io/grpc/testing/integration/RpcBehaviorLoadBalancerProviderTest.java [hunk 1]
```diff
@@ -79,12 +79,12 @@ public void parseInvalidConfig() {
...
 
   @Test
-  public void handleResolvedAddressesDelegated() {
+  public void acceptResolvedAddressesDelegated() {
     RpcBehaviorLoadBalancer lb = new RpcBehaviorLoadBalancer(new RpcBehaviorHelper(mockHelper),
         mockDelegateLb);
     ResolvedAddresses resolvedAddresses = buildResolvedAddresses(buildConfig());
-    lb.handleResolvedAddresses(resolvedAddresses);
-    verify(mockDelegateLb).handleResolvedAddresses(resolvedAddresses);
+    lb.acceptResolvedAddresses(resolvedAddresses);
```
Comment (larry-safran) [inline interop-testing/src/test/java/io/grpc/testing/integration/RpcBehaviorLoadBalancerProviderTest.java:86]: Shouldn't you be verifying the return value for this?

Comment (temawi) [reply interop-testing/src/test/java/io/grpc/testing/integration/RpcBehaviorLoadBalancerProviderTest.java]: Added.

**Review by larry-safran — APPROVED** [review]
It's not relevant in this case, but it feels like ForwardingLoadBalancer should be implementing acceptResolvedAddresses as a simple call to delegate.

**Comment by temawi** [comment]
> It's not relevant in this case, but it feels like ForwardingLoadBalancer should be implementing acceptResolvedAddresses as a simple call to delegate.

If you look at the base `LoadBalancer` class, you'll see some logic that will end up delegating to `handleResolvedAddresses()`.

FYI I have a few PRs in the works that will update some LBs and ultimately introduce the deprecation.

**Pull request merged**